### PR TITLE
Add Java file test

### PR DESCRIPTION
**Pull request by Mithil467**
Add Java file to collection of syntax highlighting text files #1213

### CODE REVIEW

File: tests/syntax-tests/source/Java/test.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import java.util.Scanner;
+
+/* This Java program was submiited to help bat
```
Comment (sharkdp) [inline tests/syntax-tests/source/Java/test.java:3]: typo

File: tests/syntax-tests/source/Java/test.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                System.out.print(ch);
+            }
+            System.out.println();
+        }
+        while(t!=0) {
+            // Decrement t
+            t -= 1;
+        }
+    }
+}
```
Comment (sharkdp) [inline tests/syntax-tests/source/Java/test.java:39]: Could you please add a trailing newline to the file? (See #975)

Comment (Mithil467) [reply tests/syntax-tests/source/Java/test.java]: Fixed!

**Comment by sharkdp** [comment]
Thank you!

**Pull request merged**